### DISCUSSION TITLE
ILLA Cloud IP public

### DISCUSSION THREAD

**Discussion by cla68x** [illacloud/illa-builder#1837] [General]
Hello, I am pretty new on Illa Cloud. I am trying to connect to an external database MySQL. I got credentials but administrator also asks for IP public for authentification. How can I provide Illa IP for connection? Thanks

**Comment by donqu1xotechina** [comment]
Hey there:

You can join our Discord channel for more support.

The solution for your issue is: Please enter the builder, click on the 'resources' tab on the top left corner, go to the resources page, click on the 'create new' button, and a popup window will appear. At the bottom of the window, there is a 'copy IP' button that allows you to copy it directly.

Best,

ILLA Team

**Comment by cla68x** [comment]
Great! found.
PS: I will move to Discord for support. Thanks

**Reply by karminski** [reply]
@cla68x hi, the public IP of Illa resource are unter the resource create panel:

![image](https://github.com/illacloud/illa-builder/assets/1567626/4cdcbe44-d949-4fff-82bb-9f28ef44f2db)

Also we have a API for this: https://peripheral-api.illasoft.com/v1/meta
